specs/merge/fork-choice.md

```diff
@@ -30,15 +30,15 @@ This is the modification of the fork choice according to the executable beacon c
 
 ```python
 class PowBlock(Container):
-    block_hash: Bytes32
+    block_hash: Hash32
     is_processed: boolean
     is_valid: boolean
     total_difficulty: uint256
 ```
 
 #### `get_pow_block`
 
-Let `get_pow_block(hash: Bytes32) -> PowBlock` be the function that given the hash of the PoW block returns its data.
+Let `get_pow_block(hash: Hash32) -> PowBlock` be the function that given the hash of the PoW block returns its data.
```

Comment:
What do you think about using `block_hash` instead of `hash`? The keyword `hash` is used in the [`hash` function](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md#hash)


```suggestion
Let `get_pow_block(block_hash: Hash32) -> PowBlock` be the function that given the hash of the PoW block returns its data.
```